src/main/java/com/powsybl/openloadflow/dc/fastdc/ConnectivityBreakAnalysis.java

```diff
@@ -115,30 +135,53 @@ private static void detectPotentialConnectivityBreak(LfNetwork lfNetwork, DenseM
                                                          List<PropagatedContingency> nonBreakingConnectivityContingencies,
                                                          List<PropagatedContingency> potentiallyBreakingConnectivityContingencies) {
         for (PropagatedContingency contingency : contingencies) {
-            List<ComputedContingencyElement> contingencyElements = contingency.getBranchIdsToOpen().keySet().stream().map(contingencyElementByBranch::get).collect(Collectors.toList());
-            if (isGroupOfElementsBreakingConnectivity(lfNetwork, states, contingencyElements, equationSystem)) { // connectivity broken
+            if (isConnectivityPotentiallyModifiedByContingencyAndOperatorStrategy(lfNetwork, states, contingency, contingencyElementByBranch, DenseMatrix.EMPTY,
+                    Collections.emptyList(), Collections.emptyMap(), equationSystem)) { // connectivity broken
                 potentiallyBreakingConnectivityContingencies.add(contingency);
             } else {
                 nonBreakingConnectivityContingencies.add(contingency);
             }
         }
     }
 
+    /**
+     * Returns true if the given contingency and operator strategy actions breaks potentially connectivity.
+     * This is determined with a "worst case" sensitivity-criterion. If the criterion is not verified, there is no connectivity break.
+     */
+    private static boolean isConnectivityPotentiallyModifiedByContingencyAndOperatorStrategy(LfNetwork lfNetwork, DenseMatrix contingencyStates, PropagatedContingency contingency,
+                                                                                             Map<String, ComputedContingencyElement> contingencyElementByBranch, DenseMatrix actionStates, List<LfAction> operatorStrategyLfActions,
+                                                                                             Map<LfAction, AbstractComputedElement> actionElementByBranch, EquationSystem<DcVariableType, DcEquationType> equationSystem) {
+        List<ComputedContingencyElement> contingencyElements = contingency.getBranchIdsToOpen().keySet().stream()
+                .map(contingencyElementByBranch::get)
+                .collect(Collectors.toList());
+        // only consider actions disabling branches to compute "worst case" sensitivity-criterion
+        // if the criterion is not verified, we are sure there is no connectivity break
```

Comment:
@p-arvy , please add more explanations on why it allows to not take line closure into account.